src/styles/_article.scss

```diff
@@ -0,0 +1,61 @@
+#article {
+  header {
+    position: relative;
+    background: $finsec-royal-blue;
+  }
+  main {
+    h2,
+    h3,
+    h4,
+    p {
+      margin-top: 20px;
+    }
+    section {
+      align-items: left;
+      margin-bottom: 40px;
+      max-width: 1260px;
+      img {
+        max-width: 100%;
+        height: auto;
+      }
+      h4 {
+        color: $finsec-grey;
+        padding-left: 100px;
+        letter-spacing: 0.5px;
+        text-transform: uppercase;
+        a {
+          color: $finsec-grey;
+        }
+        a:hover {
+          color: grey;
+        }
+      }
+      @media (max-width: 900px) {
+        h4 {
+          padding-left: 40px;
+        }
+      }
+      div {
+        color: black;
+        max-width: 600px;
+        display: flex;
+        flex-direction: column;
```

Comment:
Pro-tip: `flex-flow: column nowrap` merges both `flex-direction` and `flex-wrap`. I tend to always specify the wrap bit as well for it defines how I need the elements to behave when compressed. No need to change this though if it works in both desktop and mobile widths.